### PR TITLE
Major improvements

### LINKED ISSUE #6: Create LNL system-test pipeline
Create a pipeline that will test LNL by:
- [x] starting a network,
- [x] running scripts to change the network state,
- [x] wait for a checkpoint to be created,
- [x] stop and destroy the network,
- [x] convert checkpoint to JSON and archive it,
- [x] start a new network using checkpoint,
- [x] run scripts to validate network state,
- [x] wait for a checkpoint to be created,
- [x] convert checkpoint to JSON and archive it.

Also
- [x] create a helper function to trigger this pipeline from other pipelines as a stage.
- [ ] optionally diff botch checkpoints

### PR DESCRIPTION
**Pull request by fkondej**
Closes #6.

### CODE REVIEW

File: vars/pipelineDockerisedVega.groovy [hunk 1]
```diff
@@ -392,7 +463,7 @@ Map<String,Closure> getPrepareVegatoolsStages(
                 }
             }
         }
-        stage('Build Vegatools Docker Image') {
+        /*stage('Build Vegatools Docker Image') {
```
Comment (ashleyvega) [inline vars/pipelineDockerisedVega.groovy:466]: not building vegatools?

File: vars/pipelineSystemTestsLNL.groovy [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+import io.vegaprotocol.DockerisedVega
+
+void call() {
+    if (currentBuild.upstreamBuilds) {
+        RunWrapper upBuild = currentBuild.upstreamBuilds[0]
+        currentBuild.displayName = "#${currentBuild.id} - ${upBuild.fullProjectName} #${upBuild.id}"
+    }
+    pipelineDockerisedVega.call([
+        parameters: [
+            // TODO: wip
```
Comment (ashleyvega) [inline vars/pipelineSystemTestsLNL.groovy:14]: Is something still to be done?

File: vars/pipelineDockerisedVega.groovy (hunk 1 above)
Comment (fkondej) [reply vars/pipelineDockerisedVega.groovy:466]: Other images are required for DV. The `vegatools` is not needed for that.
Currently, the vegatools is used to convert checkpoint to JSON format. I plan to use it also to stream and logs all the events from the data-node and the vega core. But in separate ticket.

Comment (ashleyvega) [reply vars/pipelineDockerisedVega.groovy:466]: ok

File: vars/pipelineSystemTestsLNL.groovy (hunk 2 above)
Comment (fkondej) [reply vars/pipelineSystemTestsLNL.groovy]: not in here. Removed

**Review by ashleyvega — APPROVED** [review]
LGTM

**Pull request merged**